src/main/api/createAPIKey.json

```diff
@@ -0,0 +1,28 @@
+{
+  "uri": "/api/apikey/key/",
+  "comments": [
+    "Creates an authenticatio API key. You can optionally specify an Id for the application, if not provided one will be generated."
+  ],
+  "method": "post",
+  "methodName": "createAPIKey",
+  "successResponse": "APIKeyResponse",
+  "errorResponse": "Errors",
+  "params": [
+    {
+      "name": "authenticationKeyId",
+      "comments": [
+        "(Optional) The Id of the authentication key. If not provided a secure random UUID will be generated."
```

Comment:
This isn't a UUID, it is a string.  Most of our APIs use a UUID, but in this case the primary key is the API key.